src/main/kotlin/frc/robot/subsystems/superstructure/climb/ClimbSubsystem.kt

```diff
@@ -0,0 +1,52 @@
+package frc.robot.subsystems.superstructure.climb
+import com.revrobotics.CANSparkMaxLowLevel
+import org.ghrobotics.lib.commands.FalconCommand
+import org.ghrobotics.lib.mathematics.units.*
+import org.ghrobotics.lib.mathematics.units.nativeunit.*
+import org.ghrobotics.lib.motors.rev.FalconMAX
+import org.ghrobotics.lib.wrappers.FalconDoubleSolenoid
+import org.ghrobotics.lib.wrappers.FalconSolenoid
+
+object ClimbSubsystem {
+
+    class Climb(val climbHeight : SIUnit<Meter>): FalconCommand() {
+
+        //TODO Tune PID / Enter Values
+        val Kp = 0.00
+        val Kd = 0.00
+        val solenoid = FalconDoubleSolenoid(0, 1, 8)
```

Comment:
This solenoid needs to be defined within the scope of climbsubsystem rather than the command